prime-dotnet-webapi/Controllers/AdminsController.cs

```diff
@@ -54,8 +55,8 @@ public async Task<ActionResult> CreateAdmin(Admin admin)
             var createdAdminId = await _adminService.CreateAdminAsync(admin);
 
             return CreatedAtAction(
-                nameof(GetAdminById),
-                new { adminId = createdAdminId },
+                nameof(GetAdminByUserId),
+                new { userId = admin.UserId },
```

Comment:
To confirm my understanding, the return value doesn't look like it's used by the Angular code (`adjudication.guard.ts`), correct?

Reply:
Good point. It looks like the frontend does not use the additional info.